functions/vshn-postgres-func/encrypted_pvc_test.go

```diff
@@ -0,0 +1,79 @@
+package vshnpostgres
+
+import (
+	"context"
+	"testing"
+
+	xkube "github.com/crossplane-contrib/provider-kubernetes/apis/object/v1alpha1"
+	"github.com/go-logr/logr"
+	"github.com/stretchr/testify/assert"
+	vshnv1 "github.com/vshn/component-appcat/apis/vshn/v1"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/yaml"
+)
+
+func TestNoEncryptedPVC(t *testing.T) {
+	type args struct {
+		expectedFuncIO string
+		inputFuncIO    string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "GivenNoEncryptionParams_ThenExpectNoOutput",
+			args: args{
+				expectedFuncIO: "enc_pvc/01-ThenExpectNoOutput.yaml",
+				inputFuncIO:    "enc_pvc/01-GivenNoEncryptionParams.yaml",
+			},
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
```

Comment:
Table driven tests for a single test case is a bit useless. Just remove the for loop and testcase slice and test it directly

Reply:
Added a second test to test for `false` explicitly. So I guess the table driven test is now ok?